R/postgres_db_metadata.R

```diff
@@ -57,7 +57,8 @@ list_tables_with_metadata_in_statbot_db <- function() {
   metadata <- get_metadata_from_postgres()
   tables_with_table_metadata <- metadata$metadata_tables$name
   tables_with_column_metadata <- unique(metadata$metadata_table_columns$table_name)
-  tables_with_metadata <- c(tables_with_table_metadata, tables_with_column_metadata)
+  tables_with_metadata <- c(tables_with_table_metadata, tables_with_column_metadata) %>%
```

Comment:
Why did you add `unique()` here?

Reply:
because most tables have metadata in both `tables_with_table_metadata` and  `tables_with_column_metadata` so the result was that I had most table_names in there double.